tests/business/entities/test_rule_processor.py

```diff
@@ -552,7 +552,7 @@ def test_process_rules_copies_multi_coords_correctly():
     result_array = result_array.assign_coords({"test": _xr.DataArray([2, 4, 5])})
 
```

Comment:
This is enigmatic to me: 
The first line set the variable result_array_2 and the second one overwrites it again. What is the net effect? What do the numbers 1, 2 and 93 mean?